storage/src/test/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManagerMultipleSubscriptionsTest.java

```diff
@@ -49,66 +51,34 @@
 import static org.mockito.Mockito.verify;
 
 @SuppressWarnings("deprecation") // Added for Scala 2.12 compatibility for usages of JavaConverters
```

Comment:
Do we need this annotation?

Reply:
We don't need it now. Removed it. Thank you.